week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -84,7 +87,13 @@ function getTransportModes() { }
 
   Hint: Use the corresponding array method to decide if an element is member of an array.
 */
-function isAccessibleByTransportMode() { }
+function isAccessibleByTransportMode(transport, modes) {
+  if (transport.includes(modes)) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This exercise is not correct, what we expect from you is to return the index of that element if it exists in the array transport.